frontend/device/kobo/powerd.lua

```diff
@@ -106,7 +106,10 @@ function KoboPowerD:init()
         end
 
         self.isAuxBatteryConnectedHW = function(this)
-            return this:read_int_file(this.aux_batt_connected_file) == 1
+            -- aux_batt_connected_file shows us:
+            -- 0 if power cover is not connected
+            -- and 1 (or sometimes -1) if the power cover is connected (without a charger)
```

Comment:
I think this is what you were trying to say? But if not I find it confusing regardless. ;-)

```suggestion
            -- 1 if the power cover is connected
            -- -1 if the power cover is connected without a charger
```

Reply:
Yes, this is better. But the -1 will not show up every time the cover is connected without a carger :/